tests/test_metrics.py

```diff
@@ -1,6 +1,24 @@
 import numpy as np
 
-from mmseg.core.evaluation import eval_metrics, mean_dice, mean_iou
+from mmseg.core.evaluation import (eval_metrics, mean_dice, mean_fscore,
+                                   mean_iou)
+
+
+def f_score(precision, recall, beta=1):
+    """calcuate the f-score value.
+
+    Args:
+        precision (float | torch.Tensor): The precision value.
+        recall (float | torch.Tensor): The recall value.
+        beta (int): Determines the weight of recall in the combined score.
+            Default: False.
+
+    Returns:
+        [torch.tensor]: The f-score value.
+    """
+    score = (1 + beta**2) * (precision * recall) / (
+        (beta**2 * precision) + recall)
+    return score
```

Comment:
This part is the same as `mmseg.core.evaluation.metrics`. We may directly import it.